system-test/storage.js

```diff
@@ -696,6 +696,25 @@ describe('storage', function() {
     });
   });
 
+  describe('channels', function() {
```

Comment:
That would be good, but I can't create a channel without an HTTPS server. Look into how many things are required, I'm not sure I'll be able to do that for the tests :\